oss_src/sframe/sframe_rows.cpp

```diff
@@ -32,16 +34,36 @@ void sframe_rows::clear() {
 
 void sframe_rows::save(oarchive& oarc) const {
   oarc << m_decoded_columns.size();
-  for (auto& i : m_decoded_columns) oarc << *i;
+  oarchive temp_inmemory_arc;
+  for (auto& i : m_decoded_columns) {
+    v2_block_impl::block_info info;
```

Comment:
Does this only apply to sframe_rows.save()/load() or shall we move it into lower level sarray column save/load?